modules/publication/README.md

```diff
@@ -0,0 +1,45 @@
+# Publication
+
+## Purpose
+The publication module allows users to propose research projects based on LORIS 
+data, upload publication media, and allows lead study PIs to review the project 
+proposals.
+
+## Intended Users
+
+Publication module is intended for site PIs to propose specific research projects
+and for lead PIs to review them. When proposing a project, PIs may also specify 
+other LORIS users to help with editing and maintaining their project proposals.
+
+## Scope
+
+The publication module is intended to view and manage research project proposals.
+Users who are proposing a project are able to specify which specific variables of 
+interest they are interested in studying. However, gathering and releasing this 
+data remains within the scope of the DQT and Data Release modules respectively.
```

Comment:
Maybe also specify that realizing/implementing the proposal is also outside the scope of LORIS, this module only tracks the status and metadata?